ios/Mobile/MobileSocket.h

```diff
@@ -0,0 +1,31 @@
+// -*- Mode: objc; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; fill-column: 100 -*-
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#import "ios.h"
+#import <Foundation/NSObject.h>
+
+#import <WebKit/WKURLSchemeTask.h>
+#import <WebKit/WKURLSchemeHandler.h>
+
+#import "CODocument.h"
+
+#import <wsd/DocumentBroker.hpp>
+
+@interface MobileSocket : NSObject 
```

Comment:
I'd love at least a purpose comment for this interface :-) [In general - any class/interface/..., please]